CMakeLists.txt

```diff
@@ -21,20 +21,27 @@ find_package(ITK 5 REQUIRED
     ITKRegionGrowing
     ITKImageGrid
     ITKLabelMap
+    ITKRegistrationCommon
+    ITKIOTransformInsightLegacy
```

Comment:
We should use the HDF5 transform file format if possible due to its better precision and transform model support.